mlx_steering_dump_parser.py

```diff
@@ -93,16 +93,34 @@
     }
 
 
+def dr_report_unsupported_object(dump_ctx, line):
+    dr_rec_type = line[0]
+    if dr_rec_type_is_domain(dr_rec_type):
+        print("Err: Unsupported domain related object: %s" %(rec_type))
+    elif dr_rec_type_is_table(dr_rec_type):
+        print("Err: Unsupported table related object: %s" %(rec_type))
+    elif dr_rec_type_is_matcher(dr_rec_type):
+        print("Err: Unsupported matcher related object: %s" %(rec_type))
+    elif dr_rec_type_is_rule(dr_rec_type):
+        print("Err: Unsupported rule related object: %s" %(rec_type))
+    elif dr_rec_type_is_action(dr_rec_type):
+        if dump_ctx.rule:
+            action = dr_dump_action_unsupported(line)
+            dump_ctx.rule.add_action(action)
+        else:
+            print("Err: Unsupported domain related object: %s" %(rec_type))
+
+
 # parse csv record according to type and return parsed object (like dr_domain, dr_table, dr_rule ...)
-def dr_csv_get_obj(line):
+def dr_csv_get_obj(dump_ctx, line):
```

Comment:
a rule supposed to be add to dump_ctx before we parse an action, this is just a check that we do.
in case there is a mistake in the dump_file and rule in None we prevent a crash